### ISSUE TITLE
implementing quja'-layout

### ISSUE THREAD

**Issue by TheOneWithTheBraid** [florisboard/florisboard#127]
The `quja'`-keyboard layout is widely spread around Klingon speakers. Unfortunately, it seems to be difficult to implement as it features keys like `tlh`, `ng`, `ch` or `gh` which insert two or three characters at once. Moreover, not all characters are available in capital while others are *only* available in capital.

Is there any way to implement it? If I correctly understood the JSON files you describe the layouts with, you require one character per key, right?

You can see an X-Keyboard implementation of the quja'-intl-layout here: https://gitlab.com/TheOneWithTheBraid/klingon-keyboard

![Screenshot_20210108-065147_Fennec](https://user-images.githubusercontent.com/29313398/103979631-7d9b7000-517e-11eb-998d-344d3c2ece6e.png)

**Comment by patrickgold** [comment]
Yes this indeed difficult to implement. Normally the rule is 1 character per key. The exception currently is TLDs (with the code -255), but this do not lock the caps for this key either. This is definitely a more difficult thing to implement. I will add it to my todo list.

**Comment by TheOneWithTheBraid** [comment]
Well, the Caps key could simply be left out of the layout. Then, the implementation should be much easier. Klingon has absolutely no need for any modifier which is not on the quja' itself.

`q` and `Q` are the only letters which are available in both upper and lower case and there is a separated key for both cases.

Do you think `tlh` etc. are implimentable using the TLD keycode?
